presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -52,6 +52,11 @@ Array Functions
 
     Returns an array of the elements in the intersection of ``x`` and ``y``, without duplicates.
 
+.. function:: array_intersect(array(array(E))) -> array(bigint/double)
```

Comment:
Should E be `bigint/double` also?

Reply:
I copied the same format as what you did for `array_sum`.